tests/Microsoft.DotNet.Docker.Tests/RuntimeDepsImageTests.cs

```diff
@@ -73,80 +71,155 @@ public void VerifyDefaultUser(ProductImageData imageData)
             VerifyCommonDefaultUser(imageData);
         }
 
-        /// <summary>
-        /// Verifies that the packages installed in distroless images are scannable by security tools.
-        /// </summary>
         [LinuxImageTheory]
         [MemberData(nameof(GetImageData))]
-        public void VerifyDistrolessPackages(ProductImageData imageData)
+        public void VerifyDistrolessOSReleaseInfo(ProductImageData imageData)
         {
             if (!imageData.IsDistroless)
             {
-                OutputHelper.WriteLine("This test is only relevant to distroless images.");
                 return;
             }
+            Assert.NotEmpty(GetOSReleaseInfo(imageData));
+        }
 
-            const string SyftImage = "anchore/syft:v0.84.0";
-            DockerHelper.Pull(SyftImage);
+        [LinuxImageTheory]
+        [MemberData(nameof(GetImageData))]
+        public void VerifyRuntimeDepsPackages(ProductImageData imageData)
```

Comment:
The RuntimeDeps feels unncessary given the scope of the method.

```suggestion
        public void VerifyInstalledPackages(ProductImageData imageData)
```

Reply:
Since resolving https://github.com/dotnet/dotnet-docker/pull/4847#discussion_r1307411593, we can't have two methods with the same name but different signatures on RuntimeDepsImageTests. Would renaming the actual test logic method to something like `VerifyInstalledPackagesBase` work instead?

Reply:
I'm not making the connection on why method overloads don't work.  Where does this constraint come from?

Reply:
It is because I made `VerifyInstalledPackages` static so that I can call it from SdkImageTests without creating a `RuntimeDepsImageTests` instance. Can't have static and instance methods with the same name.

Reply:
Thanks, I overlooked static vs instance earlier.  How about VerifyInstalledPackagesExtra...j/k

I can't come up with anything better than VerifyInstalledPackagesBase